libs/datasets/sources/covid_tracking_source.py

```diff
@@ -100,7 +101,7 @@ def standardize_data(cls, data: pd.DataFrame) -> pd.DataFrame:
         data[cls.Fields.AGGREGATE_LEVEL] = AggregationLevel.STATE.value
         # Date checked is the time that the data is actually updated.
         # assigning the date field as the date floor of that day.
-        data[cls.Fields.DATE] = data[cls.Fields.DATE_CHECKED].dt.floor("D")
+        data[cls.Fields.DATE] = data[cls.Fields.DATE_CHECKED].dt.tz_localize(None).dt.floor("D")
```

Comment:
why is it important to localize?

Reply:
Ah because we don't use timezones in other dates, so when comparing indexes between different data sources, it doesn't work properly if the timezones are in different formats